src/OpenTelemetry/Internal/Shims/IsExternalInit.cs

```diff
@@ -0,0 +1,25 @@
+// <copyright file="IsExternalInit.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if NETFRAMEWORK || NETSTANDARD2_0_OR_GREATER
```

Comment:
NETSTANDARD2_0_OR_GREATER ->NETSTANDARD2_0 ?

Reply:
Basically this is the same as `#if NETFRAMEWORK || NETSTANDARD2_0 || NETSTANDARD2_1`. `IsExternalInit` isn't available until net5.0.

Reply:
okay got it